templates/news.html

```diff
@@ -45,7 +45,9 @@ <h1 class="site-section-title">News</h1>
                 <h2 class="site-section-meta">{{ post.title }}</h2>
                 <h2 class="site-section-sub">{{ post.date }}</h2>
                 <p>{{ post.content | safe }}</p>
-                <p><a href="{{ post.url }}">Read more..</a></p>
+                {% if post.url %}
+                  <p><a href="{{ post.url }}">Read more..</a></p>
```

Comment:
"Read more" doesn't seem quite right, because we don't have full control over the destination. Perhaps "External link:" or something would work better?